scripts/test_catalyst.sh

```diff
@@ -57,6 +57,8 @@ args=(
   "CODE_SIGN_IDENTITY=-" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"
   # GHA is still running 10.15.
   "MACOSX_DEPLOYMENT_TARGET=10.15"
+  # Run tests multiple times.
+  -test-iterations 10000
```

Comment:
It didn't flake when running 10k times yet still flakes on the nightlies on a sort of regular basis. I propose setting the test in question to have a longer timeout.

Reply:
Could it be related to running the tests for multiple platforms at the same time? Is anything in the tests related to changing and resetting external state?

Reply:
> Could it be related to running the tests for multiple platforms at the same time?

I don't think so, because i isolated the GHA so _only_ the `messaging / catalyst` was running.

> Is anything in the tests related to changing and resetting external state?

Maybe. Previously i was only running the flaky test but i will reenable all tests and try to repro one more time. If it still doesn't flake there, then I suggest extended expectation wait time and seeing if flake reoccurs in nightly

Reply:
**Update** it did not flake when running all the `messaging` tests. I suggest extending expectation wait time and investigating further if it flakes in nightly again.